apps/client/src/pages/home/Home.page.tsx

```diff
@@ -33,6 +35,20 @@ export function Home(): ReactElement {
   };
 
   const [userName, setUserName] = useState<string>('');
+  const [avtarURL, setAvtarURL] = useState<string>('');
+
+  useEffect(() => {
+    if (userName === '') {
+      setAvtarURL('');
+    } else {
+      setAvtarURL(generateAvatarLink(userName));
+    }
+
+    // return () => {
```

Comment:
What's up with the commented code?

Reply:
Fixed 0cd6876cdd72a5bad6d8dfd6e0ada2eca2449f42